### ISSUE TITLE
PROD: captions with code in them are broken

### ISSUE THREAD

**Issue by AllenDowney** [AllenDowney/ThinkComplexity2#44]
It looks like when I have a code snippet in a caption, the typesetting of the caption gets messed up.  There's an example on page 119.

Same issue comes up in the footnote on page 124.

Is the DocBook I am generating correct, and getting rendered wrong, or is there something wrong with it?

**Comment by kristenORM** [comment]
It looks like the tags don't enclose the text they are supposed to (though I'm not sure why your DocBook is doing this). Take this example from Figure 2-4:

``<literal` remap="verb" moreinfo="none">n=10 and <literal remap="verb" moreinfo="none"/></literal>p=0.3``

I'm getting the copyedit file ready this morning, and will correct these as follows:
`<literal remap="verb" moreinfo="none">n=10</literal> and <literal remap="verb" moreinfo="none">p=0.3</literal>`

**Comment by AllenDowney** [comment]
Ok, thanks.  I will look into fixing this in my translator, but since there are
not very many instances, it might be easier to fix them by hand.

**Comment by kristenORM** [comment]
Yeah, I have already fixed the figure captions by hand in Atlas (we've split the book file into individual chapter XML docs). Going to make a pass through the PDF to see if there are any others like the footnote you mentioned, but if I miss them, we can certainly fix by hand later.

**Comment by AllenDowney** [comment]
Sounds good.  We can catch them in QC if you want to hold off.
